journalism_workflow/static/journalism_workflow/reporter/partials/reporter.html

```diff
@@ -0,0 +1,82 @@
+<section class="section-panel">
+  <div class="container-fluid">
+    <div class="row section-header">
+      <div class="col-lg-12 col-md-12 col-sm-12">
+        <h3>
+	  Draft the article based on the editor's story plan.
```

Comment:
I'm absolutely fine with this, though you mentioned wanting to have different instructions for reviewers?